server/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const port = 3002;
+const {db, Reviews} = require('./db');
+
+app.get('/', (req, res) => res.send('YUP'));
+
+app.get('/reviews', (req, res) => {
+  Reviews.find()
+    .then((data) => res.send(data))
+    .catch((err) => res.send(err));
```

Comment:
Never send your error directly to the end user as you're revealing a lot of clues about your server (which you never want to do due to security concerns).

Reply:
I have responded with a 500 status code and an error string instead of the actual error.